### DISCUSSION TITLE
rotate Octagram to be a Brightness icon

### DISCUSSION THREAD

**Discussion by Mariusthvdb** [Mariusthvdb/custom-icons#29] [Icon suggestions/requests]
see: https://github.com/Templarian/MaterialDesign/issues/6628#issuecomment-1151102018

and more elaborate the linked discussion in https://github.com/Templarian/MaterialDesign/issues/6433

> 100% dark. Just to be clear here, this would actually mean 100% bright, and that's indicated by using brightness-7. If you are looking for just a simple octagram, we also have that as octgram, but if you need it to match the other brightness' orientation, you'll need to rotate the icon 12.5%.

needing those for some illuminance sensors, with some more granularity than currently available.

<img width="507" alt="Schermafbeelding 2022-06-09 om 23 49 24" src="https://user-images.githubusercontent.com/33354141/172951187-c0f1665a-180d-44b4-9e89-e31c9d23a4ee.png">

not onlyis the icon rotated, it also seems smaller?

An alert icon would be very good too

**Comment by arallsopp** [comment]
Does this help you? 
<a href="https://user-images.githubusercontent.com/2462104/175989262-b56ac8e3-5fe9-411a-bf9c-91d63336c2fd.svg"><img src = "https://user-images.githubusercontent.com/2462104/175989262-b56ac8e3-5fe9-411a-bf9c-91d63336c2fd.svg" width="24px"></a> 24px, based upon the existing brightness-6, but without the cut-out.

**Reply by arallsopp** [reply]
How would you have the interim values?

1. By moving a vertical bar (per the 50/50 laundry icon above)
2. Shading around a clock face (per the [linked request](https://github.com/Templarian/MaterialDesign/issues/6433)), or like [this video](https://user-images.githubusercontent.com/2462104/175996828-292bbeb0-eff9-435f-b0b3-d8ecbbcebb69.mov).
3. Eclipsing a crescent like the MDI standard per [this video](https://user-images.githubusercontent.com/2462104/175993251-2a468f3d-f51e-45d0-9bae-1e80a02c8d7c.mov)

I've got the illustrator files, so just nominate.

**Reply by Mariusthvdb** [reply]
can not view the videos, so need to try in a another browser maybe.

since yo ask ;-) I dont particularly like the 'laundry' icon, as it seems to be a half moon if you ask me. If at all possible, I would say going from full to empty in a diminishing fill from center would be optimal. not sure how many would still be discernible ?

Een library would not fir in that series I guess, as it simply has a circle inside a circle.

what would be your thoughts?

and yes, the full icon posted does help! thanks for that

**Reply by Mariusthvdb** [reply]
was able to open the videos in Chrome..

not the eclipse, that resembles the moon too much.

the clock face seems better, though might indeed be too much of a clock

pulsating circle is the best I can describe it I think https://www.istockphoto.com/nl/search/2/film?phrase=pulsating+circle starting out empty, next a small dot in the middle growing bigger until it fully fills the circle.

or, starting out empty, filling from the outer circle growing to the center...

hope you understand what I mean...

**Reply by arallsopp** [reply]
Its funny. When you start trying to describe something visual, it becomes quickly apparent how imprecise the written language is.
Do you mean like this?

https://user-images.githubusercontent.com/2462104/176197806-6d80e60d-3e2f-4459-8cbd-0b6bed38bf34.mp4

**Reply by Mariusthvdb** [reply]
yes!! magician!

that or the negative of it, not sure which would be most striking
cool.

**Reply by arallsopp** [reply]
<table>
    <tr>
        <td><a href="https://user-images.githubusercontent.com/2462104/176216068-db9d5eea-64c5-456b-9cf6-d0de3c9b9525.svg"><img src="https://user-images.githubusercontent.com/2462104/176216068-db9d5eea-64c5-456b-9cf6-d0de3c9b9525.svg" width="24px"/></a></td>
        <td><a href="https://user-images.githubusercontent.com/2462104/176216069-558afec8-1f6e-4fa8-bef6-7359d65c2f9e.svg"><img src="https://user-images.githubusercontent.com/2462104/176216069-558afec8-1f6e-4fa8-bef6-7359d65c2f9e.svg" width="24px"/></a></td>
        <td><a href="https://user-images.githubusercontent.com/2462104/176216070-391972dd-7306-4055-a2ec-bdf6d68cda90.svg"><img src="https://user-images.githubusercontent.com/2462104/176216070-391972dd-7306-4055-a2ec-bdf6d68cda90.svg" width="24px"/></a></td>
        <td><a href="https://user-images.githubusercontent.com/2462104/176216073-49f46839-355e-46a7-b19c-280984a96dbc.svg"><img src="https://user-images.githubusercontent.com/2462104/176216073-49f46839-355e-46a7-b19c-280984a96dbc.svg" width="24px"/></a></td>
        <td><a href="https://user-images.githubusercontent.com/2462104/176216076-69e3313d-5126-4119-82b6-c9e8b4e75900.svg"><img src="https://user-images.githubusercontent.com/2462104/176216076-69e3313d-5126-4119-82b6-c9e8b4e75900.svg" width="24px"/></a></td>
        <td><a href="https://user-images.githubusercontent.com/2462104/176216077-c5413bf1-cc70-44b9-a2d0-a6838d1b05c1.svg"><img src="https://user-images.githubusercontent.com/2462104/176216077-c5413bf1-cc70-44b9-a2d0-a6838d1b05c1.svg" width="24px"/></a></td>
        <td><a href="https://user-images.githubusercontent.com/2462104/176216080-135c6e64-65e9-4e2d-aa40-c6268ce7e25d.svg"><img src="https://user-images.githubusercontent.com/2462104/176216080-135c6e64-65e9-4e2d-aa40-c6268ce7e25d.svg" width="24px"/></a></td>
        <td><a href="https://user-images.githubusercontent.com/2462104/176216082-5d0ebde0-f1f0-40ca-933a-04b668ef3d00.svg"><img src="https://user-images.githubusercontent.com/2462104/176216082-5d0ebde0-f1f0-40ca-933a-04b668ef3d00.svg" width="24px"/></a></td>
        <td><a href="https://user-images.githubusercontent.com/2462104/176216084-644dfb9b-8856-45fd-870d-0a30a5cb56c9.svg"><img src="https://user-images.githubusercontent.com/2462104/176216084-644dfb9b-8856-45fd-870d-0a30a5cb56c9.svg" width="24px"/></a></td>
        <td><a href="https://user-images.githubusercontent.com/2462104/176216085-82ccbce7-5826-447e-9916-099d6ab9ff22.svg"><img src="https://user-images.githubusercontent.com/2462104/176216085-82ccbce7-5826-447e-9916-099d6ab9ff22.svg" width="24px"/></a></td>
        <td><a href="https://user-images.githubusercontent.com/2462104/176216064-581e7d4b-2770-4b78-a02b-8feadb32dcd4.svg"><img src="https://user-images.githubusercontent.com/2462104/176216064-581e7d4b-2770-4b78-a02b-8feadb32dcd4.svg" width="24px"/></a></td>
    </tr>
    <tr>
        <td>0</td>
        <td>1</td>
        <td>2</td>
        <td>3</td>
        <td>4</td>
        <td>5</td>
        <td>6</td>
        <td>7</td>
        <td>8</td>
        <td>9</td>
        <td>10</td>
    </tr>
</table>

Each is clickable to the full size version. I didn't clear up the illustrator files, so you might want to cull some of the descriptors from the svgs. Chances are, you're putting this into a js file as a path and that won't matter. Ping me if you need someone to convert the circles to paths. Should just be a case of making each a compound path, then resaving.

Oh, and they're probably numbered backwards. The darkest one is probably really the lightest, given how they're used. Have never worked this out. Same is true in [hass-hue-icons](https://github.com/arallsopp/hass-hue-icons), where the 'lit' bit of the light shows as dark when the light is coloured, eg:
<img width="583" alt="image" src="https://user-images.githubusercontent.com/2462104/176219783-ef6a2f60-c9a1-4a81-90c8-65b2bd64bd91.png">

**Reply by Mariusthvdb** [reply]
10 should suffice ;-)
and dark would be for no light in my thinking, and 10 would be as bright as it gets .... so numbered correctly ?

**Reply by arallsopp** [reply]
I updated my comment above to try to illustrate what I mean about light and dark. I'm happy to agree with you, though. Your repo, and you have a way better understanding of MDI standards than I.

**Reply by Mariusthvdb** [reply]
comparing 9 and 10 here, and the outer circle on  9 is bigger than on 10?

**Reply by arallsopp** [reply]
I should add, I’m currently battling covid and jet lag, and that I just about have the visual acuity to pick myself out of a line up.*


* if I’m wearing a name badge.

**Reply by Mariusthvdb** [reply]
o dear. take care!

**Comment by Mariusthvdb** [comment]
guess it would become
![brightness-10](https://user-images.githubusercontent.com/33354141/176221990-9396da83-b4ec-4ae0-8802-eee8c75cdd01.svg)

**Reply by Mariusthvdb** [reply]
is that inner circle the same dimension as the current brightness-5?
seems a bit 'fatter', but that might be an optical illusion

**Reply by Mariusthvdb** [reply]
![brightness](https://user-images.githubusercontent.com/33354141/176254584-9b00537f-892a-439a-bd53-98638328c5f7.gif)

**Reply by arallsopp** [reply]
Yep. You're probably right. Ok. I think I know what we are trying to do. I think I'll have to wait until I'm functioning properly before I can confidently get this right. The size of the cut out was based upon https://materialdesignicons.com/icon/brightness-6, as that was the one that I felt indicated varying brightness. From the above, it looks like brightness-5 cut out is a bit smaller (assuming that's the one for brightness-10 in the video above?).

If we are doing a smaller cut out, do you think we might need to cut the number of steps down, else they'll be really marginal and below a single pixel in width.

**Reply by Mariusthvdb** [reply]
tbh, I would love the cutout of 10 to be just like the others, so a bit bigger?

it would then also fit the -alt series nicely as closing icon

**Comment by Mariusthvdb** [comment]
would it also be possible to do the negative, start full black and then a tiny white spot, growing to the full open?

**Reply by arallsopp** [reply]
<table>
    <tr>
<td><a href="https://user-images.githubusercontent.com/2462104/176233708-37c73c5a-0a74-452f-8c4c-9fe6e95e4385.svg"><img src="https://user-images.githubusercontent.com/2462104/176233708-37c73c5a-0a74-452f-8c4c-9fe6e95e4385.svg" width="24px"/></a></td>
<td><a href="https://user-images.githubusercontent.com/2462104/176233713-cc31d83e-d6f1-4e18-826e-d88134b3d1c8.svg"><img src="https://user-images.githubusercontent.com/2462104/176233713-cc31d83e-d6f1-4e18-826e-d88134b3d1c8.svg" width="24px"/></a></td>
<td><a href="https://user-images.githubusercontent.com/2462104/176233715-94146078-d91d-4bb7-bbf0-57e900b3c060.svg"><img src="https://user-images.githubusercontent.com/2462104/176233715-94146078-d91d-4bb7-bbf0-57e900b3c060.svg" width="24px"/></a></td>
<td><a href="https://user-images.githubusercontent.com/2462104/176233716-cd537c6d-4248-49b2-b793-263be84836a6.svg"><img src="https://user-images.githubusercontent.com/2462104/176233716-cd537c6d-4248-49b2-b793-263be84836a6.svg" width="24px"/></a></td>
<td><a href="https://user-images.githubusercontent.com/2462104/176233717-ff76439b-8d8c-4df0-b59e-c4587b80e633.svg"><img src="https://user-images.githubusercontent.com/2462104/176233717-ff76439b-8d8c-4df0-b59e-c4587b80e633.svg" width="24px"/></a></td>
<td><a href="https://user-images.githubusercontent.com/2462104/176233720-111808a3-a1dc-4567-aa7c-e370da4adc10.svg"><img src="https://user-images.githubusercontent.com/2462104/176233720-111808a3-a1dc-4567-aa7c-e370da4adc10.svg" width="24px"/></a></td>
<td><a href="https://user-images.githubusercontent.com/2462104/176233721-49cdadf2-6170-4a97-80b3-bf442fa1aa2e.svg"><img src="https://user-images.githubusercontent.com/2462104/176233721-49cdadf2-6170-4a97-80b3-bf442fa1aa2e.svg" width="24px"/></a></td>
<td><a href="https://user-images.githubusercontent.com/2462104/176233723-f2ea5d7e-bfaa-4f38-bc8d-168487d9ed81.svg"><img src="https://user-images.githubusercontent.com/2462104/176233723-f2ea5d7e-bfaa-4f38-bc8d-168487d9ed81.svg" width="24px"/></a></td>
<td><a href="https://user-images.githubusercontent.com/2462104/176233724-bd9457c9-23cf-47c4-82ab-d2dba3f8ef3a.svg"><img src="https://user-images.githubusercontent.com/2462104/176233724-bd9457c9-23cf-47c4-82ab-d2dba3f8ef3a.svg" width="24px"/></a></td>
<td><a href="https://user-images.githubusercontent.com/2462104/176233726-3a81fa89-5395-4346-9b7c-366f368ae2dc.svg"><img src="https://user-images.githubusercontent.com/2462104/176233726-3a81fa89-5395-4346-9b7c-366f368ae2dc.svg" width="24px"/></a></td>
    </tr>
    <tr>
        <td>10</td>
        <td>9</td>
        <td>8</td>
        <td>7</td>
        <td>6</td>
        <td>5</td>
        <td>4</td>
        <td>3</td>
        <td>2</td>
        <td>1</td>
    </tr>
</table>

**Reply by Mariusthvdb** [reply]
Yesss!

**Reply by Mariusthvdb** [reply]
same question, (were missing out on 1 icon (10) btw compared to the first series..)

![brightness-alt](https://user-images.githubusercontent.com/33354141/176254918-59fa88d8-6438-494b-bafb-0c50aa1f10be.gif)
